src/routes/safe/components/Transactions/TxList/TxSummary.tsx

```diff
@@ -33,54 +89,28 @@ export const TxSummary = ({ txDetails }: Props): ReactElement => {
           <TxShareButton id={txDetails.txId} />
         </div>
       )}
-      <div className="tx-hash">
-        <Text size="xl" strong as="span">
-          Transaction hash:{' '}
-        </Text>
-        {txHash ? (
-          <InlineEthHashInfo textSize="xl" hash={txHash} shortenHash={8} showCopyBtn explorerUrl={explorerUrl} />
-        ) : (
-          <Text size="xl" as="span">
-            {NOT_AVAILABLE}
-          </Text>
-        )}
-      </div>
-      {safeTxHash !== undefined && (
-        <div className="tx-hash">
-          <Text size="xl" strong as="span">
-            SafeTxHash:{' '}
-          </Text>
-          <InlineEthHashInfo textSize="xl" hash={safeTxHash} shortenHash={8} showCopyBtn />
-        </div>
-      )}
-      {nonce !== undefined && (
-        <div className="tx-nonce">
-          <Text size="xl" strong as="span">
-            Nonce:{' '}
-          </Text>
-          <Text size="xl" as="span">
-            {nonce}
-          </Text>
-        </div>
+      {txHash && <TxDataRow title="Transaction hash:" value={txHash} inlineType="hash" />}
```

Comment:
Noice refactoring!

Reply:
Why remove the class names though?

Reply:
It might be used in E2E tests?

Reply:
I've just checked. More, it would have broken the E2E in the PR.